engine/apps/slack/scenarios/distribute_alerts.py

```diff
@@ -128,26 +129,102 @@ def process_signal(self, alert: Alert) -> None:
         if num_updated_rows == 1:
             # this will be the case in the event that we haven't yet created a Slack message for this alert group
 
-            slack_channel = (
-                alert_group.channel_filter.slack_channel
-                if alert_group.channel_filter
-                # if channel filter is deleted mid escalation, use default Slack channel
-                else alert_receive_channel.organization.default_slack_channel
-            )
+            # if channel filter is deleted mid escalation, use the organization's default Slack channel
+            slack_channel = channel_filter.slack_channel if channel_filter else organization.default_slack_channel
+
+            # slack_channel can be None if
+            # - the channel filter is deleted mid escalation
+            # - AND the organization does not have a default slack channel set
```

Comment:
If I'm not mistaken, `channel_filter.slack_channel` also can be None if slack channel was not specify for the channel filter :thinking: We should probably use `slack_channel_or_org_default` here instead:
```suggestion
            slack_channel = channel_filter.slack_channel_or_org_default if channel_filter else organization.default_slack_channel

            # slack_channel can be None if
            # - the channel filter is deleted mid escalation
            # - AND the organization does not have a default slack channel set
```